### PR TITLE
Get rid of all annoying chrome callouts

### PR DESCRIPTION
**Pull request by renajohn**

### CODE REVIEW

File: agent/wptdriver/wpt_settings.cc [hunk 1]
```diff
@@ -56,6 +56,9 @@ static const TCHAR * CHROME_REQUIRED_OPTIONS[] = {
   _T("disable-component-update"),
   _T("disable-background-downloads"),
   _T("ignore-certificate-errors"),
+  _T("test-type"),
```
Comment (ocr) [inline agent/wptdriver/wpt_settings.cc:59]: What is this option?

Comment (renajohn) [inline agent/wptdriver/wpt_settings.cc:59]: To tell chrome we are in test mode so `ignore-certificate-errors` is ok.

Comment (renajohn) [inline agent/wptdriver/wpt_settings.cc:59]: https://bugs.chromium.org/p/chromium/issues/detail?id=376424

**Comment by ocr** [comment]
:+1: :+1:

**Pull request merged**